usr.sbin/sysinst/msg.mbr.pt

```diff
@@ -0,0 +1,140 @@
+/*	$NetBSD: msg.mbr.pt,v 1.4 2019/06/12 06:20:17 martin Exp $	*/
+
+/*
+ * Copyright 1997 Piermont Information Systems Inc.
+ * All rights reserved.
+ *
+ * Written by Philip A. Nelson for Piermont Information Systems Inc.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ * 3. The name of Piermont Information Systems Inc. may not be used to endorse
+ *    or promote products derived from this software without specific prior
+ *    written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY PIERMONT INFORMATION SYSTEMS INC. ``AS IS''
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL PIERMONT INFORMATION SYSTEMS INC. BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+ * THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+/* MBR Message catalog -- Portuguese, i386 version */
+
+/* NB: Lines ending in spaces force line breaks */
+
+message mbr_part_header_1	{Tipo}
+message mbr_part_header_2	{Montar}
+.if BOOTSEL
+message mbr_part_header_3	{Menu de inicializa�ao}
+.endif
+
+message noactivepart
+{Voc� nao marcou uma parti�ao como ativa. Isso pode fazer com que o seu 
+sistema nao inicie corretamente.}
+
+message fixactivepart
+{
+A parti�ao NetBSD do disco deve ser marcada como ativa?}
+
+message setbiosgeom
+{
+Voc� ser� solicitadoa informar a geometria.
+Insira o n�mero de stores por trilha (m�ximo 63)
+e o n�mero de cabe�alhos (m�ximo 256) queo BIOS usa para acessar o disco.
+O n�mero de cilindros ser� calculado a partir do tamanho do disco.
+
+}
+
+message nobiosgeom
+{Sysinst nao p�de determinar automaticamente a geometria do BIOS do disco.
+A geometria f�sica � %d cilindros %d setores %d cabe�alhos\n}
+
+message biosguess
+{Usando as informa�oes existente no disco, meu melhor palpite para a geometria
+do BIOS � %d cilindros %d setores %d cabe�alhos\n}
+
+message realgeom
+{real geom: %d cilindros, %d cabe�alhos, %d setores  (OBS: apenas para compara�ao)\n}
+
+message biosgeom
+{BIOS geom: %d cilindros, %d cabe�alhos, %d setores\n}
```

Comment:
remark: I would say that, for this context, the correct translation for "head" would be "cabeças", not "cabeçalhos". But that's just my opinion man

Reply:
Yeah, I thought the term 'cabeçalho' could be better, but looking at some online documentation about Hard Drivers, the expression is really: 'cabeça'. I'll change it.